controllers/permits/applications/special-uses/commercial/temp-outfitters/index.js

```diff
@@ -119,103 +118,166 @@ function fileErrors(missingFiles){
 	output = output.trim();
 	return output;
 }
+
+function concatErrors(errorMessages){
```

Comment:
Also, just looking at code coverage, this function is not being tested at all. Creating a function in util that buildErrorMessage also uses will help up the function coverage.